client/state/analytics/reducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+
+export const isTracking = ( state = false, { type } ) => {
```

Comment:
> Like I said, I thought predicates like isDoingSomething were fine for selectors (which are read-only, so to speak), and I thought a different nomenclature for reducers was good to differentiate them easily.

I'm all for removing unnecessary state altogether and I think we can usually remove boolean values and rely on other existing data, such as `null` values instead of empty values.

On that note, why should the reducers and selectors need different names? I'm uncomfortable saying they need to be the same, but it feels at least as bad if not worse to suggest that they shouldn't match, after all, the selectors are simply there to free up the internal representation of the data in the reducers, right, not to communicate different information altogether (barring the computed selectors)?